spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/TaskServiceUtils.java

```diff
@@ -237,6 +238,29 @@ private static String updateProperties(TaskNode taskNode, TaskApp subTask, Map<S
 		return result;
 	}
 
+	private static String updateVersionProperties(TaskNode taskNode, TaskApp subTask, Map<String, String> taskDeploymentProperties,
+			String result, String prefix) {
```

Comment:
Since updateVersionProperties only has one prefix we can probably make it a constant in the method.